### PR TITLE
Add script to reset retention period

### PR DESCRIPTION
**Pull request by tarnap**
See https://extranet.4teamwork.ch/support/gemeinde-koeniz/tracker/25

### CODE REVIEW

File: opengever/maintenance/scripts/reset_retention_period.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+class RetentionPeriodResetter(object):
+    """Resets the retention periods
+    """
+
+    def __init__(self, plone, options):
+        self.plone = plone
+        self.options = options
+        self.portal_setup = api.portal.get_tool('portal_setup')
```
Comment (phgross) [inline opengever/maintenance/scripts/reset_retention_period.py]: ✂️

**Comment by tarnap** [comment]
@phgross: Please re-review

File: opengever/maintenance/scripts/reset_custody_period.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        self.plone = plone
+        self.options = options
+        self.catalog = api.portal.get_tool('portal_catalog')
+
+    def run(self):
+        for item in self.catalog.unrestrictedSearchResults(
+                object_provides=[ILifeCycleMarker.__identifier__]):
+            obj = item.getObject()
+
+            ILifeCycle(obj).custody_periods = u'0'
```
Comment (phgross) [inline opengever/maintenance/scripts/reset_custody_period.py]: need to be int.

**Comment by tarnap** [comment]
@phgross: I changed the type from unicode to integer. Please re-review

**Pull request merged**